src/index.html

```diff
@@ -11,8 +11,46 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__imac">
+        <img
+          src="images/imac.jpeg"
+          alt="imac"
+        />
+      </div>
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__subtitle">Product code: 195434</p>
+      <div class="card__review">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="card__review-label">Reviews: 5</p>
```

Comment:
Ensure that the 'stars' block is implemented as a separate BEM block so it can be reused in other places if necessary. The modifier class 'stars--4' should indicate that 4 stars are active (e.g. by changing their color), and it should be implemented in the corresponding CSS file.